dask_ml/model_selection/_hyperband.py

```diff
@@ -384,8 +384,7 @@ def _get_SHAs(self, brackets):
             SHAs[b] = sha
         return SHAs
 
-    @gen.coroutine
-    def _fit(self, X, y, **fit_params):
+    async def _fit(self, X, y, **fit_params):
```

Comment:
Can you make these changes in a dedicated PR?

Reply:
Yeah, I can do that. I'll try to get it done today, but no promises.

Reply:
See #675